### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by Arterter** [SajjadArifGul/PasswordManager#1]
I need help on visual studio i am getting so much errors. pls help me

**Comment by Arterter** [comment]
Severity	Code	Description	Project	File	Line	Suppression State
Error		This project references NuGet package(s) that are missing on this computer. Use NuGet Package Restore to download them.  For more information, see http://go.microsoft.com/fwlink/?LinkID=322105. The missing file is ..\packages\System.Data.SQLite.Core.1.0.105.0\build\net451\System.Data.SQLite.Core.targets.	PasswordManager.Database	C:\Users\André Silva\Downloads\PasswordManager-master\PasswordManager.Database\PasswordManager.Database.csproj	112	




I don't know how to put it working

**Comment by SajjadArifGul** [comment]
delete packages folder in repository and open Nuget Packages Manager, go to Updates tab and update all the nuget packages.

**Comment by Arterter** [comment]
![image](https://user-images.githubusercontent.com/36677539/57221613-cd17b900-6ff7-11e9-8acd-8444b41f79e1.png)

Ty for your replay but that doesn't work because the program wants the Version 1.105 but if i update we updates to 1.110

Am I doing right ??

**Comment by Arterter** [comment]
ty for you replying, I already tried that but doesn't work, I have posted
in same issue thread, I took a print of that.

I am very thankful.

**Comment by SajjadArifGul** [comment]
Yes you did the right steps. I think its an issue with your visual studio. Reload the project and then run it. Itll work.

**Comment by Arterter** [comment]
Ty for your reply, i click on rebuild is that ??? 

![image](https://user-images.githubusercontent.com/36677539/57246234-aaf05c00-7034-11e9-8534-da8e77878501.png)

i start project and i get sqlite.interop.dll error when i try to create an user


**EDIT:*** i find the reload button i did that, but i am havving same error

**Comment by SajjadArifGul** [comment]
You are missing Sqlite.Interop.dll file. Download it from SQlite website and add it to your bin folder. Also check this link: https://rashimuddin.wordpress.com/tag/sqlite-interop-dll/

**Comment by Arterter** [comment]
Hi, sry for my late reply but i am in different country timezone.

I already installed sqlite-netFx451-setup-bundle-x64-2013-1.0.110.0  and i got this folder.

![image](https://user-images.githubusercontent.com/36677539/57292261-44f9e800-70b9-11e9-89ca-f337568a688f.png)

and on my visual basic I did what the blog says **copy the System.Data.SQLite.dll to my project folder and refer this dll to my project.** , but I don't know if it is right

![image](https://user-images.githubusercontent.com/36677539/57292497-be91d600-70b9-11e9-94ac-a284db49bda5.png)

I copy the 3 dll from C:\Program Files\System.Data.SQLite\2015\bin to a folder in my project folder and then i add on reference 

![image](https://user-images.githubusercontent.com/36677539/57292732-8b9c1200-70ba-11e9-9519-939e91d7badd.png)

and then I install via Package Manager Console the sqlite using Install-Package System.Data.SQLite

after that i didn't get the x86 and x64 folder with the SQLite.Interomp.dll like the vlog says.

TY for your affort
